task_spec/spec/acceptance/init_spec.rb

```diff
@@ -22,7 +22,7 @@ def inventory
 
   it 'works with version and install tasks' do
     # test the agent isn't already installed and that the version task works
-    results = run_task('puppet_agent::version', 'default', config: config, inventory: inventory)
```

Comment:
Changing from `default` to `target` allows multiple targets defined in hosts.yaml to be tested. 
**For example**: 
1. Generate multiple hosts
`beaker-hostgenerator sles11-64target.a-sles12-64target.a > hosts.yaml`
hosts.yaml
```
---
HOSTS:
  sles11-64-1:
    pe_dir: 
    pe_ver: 
    pe_upgrade_dir: 
    pe_upgrade_ver: 
    hypervisor: vmpooler
    platform: sles-11-x86_64
    packaging_platform: sles-11-x86_64
    template: sles-11-x86_64
    roles:
    - agent
    - target
  sles12-64-1:
    pe_dir: 
    pe_ver: 
    pe_upgrade_dir: 
    pe_upgrade_ver: 
    hypervisor: vmpooler
    platform: sles-12-x86_64
    packaging_platform: sles-12-x86_64
    template: sles-12-x86_64
    roles:
    - agent
    - target
CONFIG:
  nfs_server: none
  consoleport: 443
  pooling_api: http://vmpooler.delivery.puppetlabs.net/
```
2. Run task acceptance tests against targets defined in hosts.yaml with `BEAKER_setfile`
`GEM_BOLT=true BEAKER_keyfile=~/.ssh/id_rsa-acceptance BEAKER_debug=true BEAKER_setfile=hosts.yaml bundle exec rake task_acceptance`

When `default` is targeted instead of `target` the following error occurs:
```
     Failure/Error: expect(res).to include('status' => 'success')
       expected {"node" => "default", "result" => {"_error" => {"details" => {}, "issue_code" => "CONNECT_ERROR", "kind" => "pupp...msg" => "Failed to connect to default: getaddrinfo: Name or service not known"}}, "status" => "failure"} to include {"status" => "success"}
```